From c7f10fa85dd50b7270d328f601bc61e46eb17b75 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Thu, 3 May 2012 11:03:21 +0800
Subject: [PATCH 1413/2463] ENGR00181196-2 ov5642: fix the first frame of
 capture quality is bad

Add 300ms delay after intial setting is download

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 drivers/media/video/mxc/capture/ov5642.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ov5642.c b/drivers/media/video/mxc/capture/ov5642.c
index f8f8a6c..d6dad5f 100644
--- a/drivers/media/video/mxc/capture/ov5642.c
+++ b/drivers/media/video/mxc/capture/ov5642.c
@@ -279,7 +279,7 @@ static struct reg_value ov5642_initial_setting[] = {
 	{0x5007, 0x00, 0, 0}, {0x5009, 0x00, 0, 0}, {0x5011, 0x00, 0, 0},
 	{0x5013, 0x00, 0, 0}, {0x519e, 0x00, 0, 0}, {0x5086, 0x00, 0, 0},
 	{0x5087, 0x00, 0, 0}, {0x5088, 0x00, 0, 0}, {0x5089, 0x00, 0, 0},
-	{0x302b, 0x00, 0, 0},
+	{0x302b, 0x00, 0, 300},
 };
 
 static struct reg_value ov5642_setting_15fps_QCIF_176_144[] = {
-- 
1.7.7.4

